browser-test/src/admin_application_statuses.test.ts

```diff
@@ -54,14 +54,15 @@ describe.skip('view program statuses', () => {
       await logout(pageObject)
     })
 
-    it('does not Show status options', async () => {
+    it('does not show status options or edit note', async () => {
       await loginAsProgramAdmin(pageObject)
 
       await adminPrograms.viewApplications(programWithoutStatusesName)
 
       await adminPrograms.viewApplicationForApplicant(userDisplayName())
 
       expect(await adminPrograms.isStatusSelectorVisible()).toBe(false)
```

Comment:
related to the discussion we had on the previous PR, I think the setup here should be in the `describe` `beforeAll`, and then each expect in its own `it`.  Then they can fail individually and we get clear feedback around which is failing, and without this selector failure shadowing if the Edit Note is working or not.